### ISSUE TITLE
restservice-spark: Unable to find ServletContextHandler for provided ServletContext

### ISSUE THREAD

**Issue by ts-00** [microstream-one/microstream#413]
## Environment Details
* MicroStream Version: 07.00.00-MS-GA
* JDK version: 17
* Used frameworks: Wildfly Server (version 26)

## Describe the bug
I used Wildfly with `Microstream version 06.00.00-MS-GA` without any problems. I only needed to use `spark-core v 2.8.0` instead of  `spark-core v 2.9.3` because of https://github.com/microstream-one/microstream/issues/238. 

After updating my `pom.xml` to use `Microstream version 07.00.00-MS-GA`, I get the following error:

```
Caused by: java.lang.RuntimeException: java.lang.IllegalStateException: Unable to find ServletContextHandler for provided ServletContext
	at io.undertow.servlet@2.2.14.Final//io.undertow.servlet.core.DeploymentManagerImpl.deploy(DeploymentManagerImpl.java:257)
	at org.wildfly.extension.undertow@26.0.1.Final//org.wildfly.extension.undertow.deployment.UndertowDeploymentService.startContext(UndertowDeploymentService.java:105)
	at org.wildfly.extension.undertow@26.0.1.Final//org.wildfly.extension.undertow.deployment.UndertowDeploymentService$1.run(UndertowDeploymentService.java:87)
	... 8 more
```

The error above is like https://github.com/microstream-one/microstream/issues/238.

Comparing the dependency hierarchy between `Microstream v6` and `Microstream v7` shows that `microstream-storage-restservice-sparkjava` of `Microstream v7`now uses libs with other versions as the libs within `spark-core v 2.8.0`. In `Microstream v6`, Microstream used the libs from the `spark-core v 2.8.0` instead:

![dep-ms-7](https://user-images.githubusercontent.com/80713300/186147295-f9318f3e-304e-4a86-946e-9544cbb0bcd0.png)

To sum up: 
- In `Microstream v6` the `jetty-libs` of the provided `spark-lib` was used which leads to _only_ one workaround for the user (by using `spark v2.8.0` instead of `spark v2.8.3`).
-  `Microstream v7` needs the `spark-lib` and additionally it uses other `jetty-libs` than the `spark-lib` which causes something like https://github.com/perwendel/spark/issues/1203. Because of this, I need to make 2 workarounds. 

Maybe microstream (`microstream-storage-restservice-sparkjava`) can provide a better solution by not using or sharing libs which conflicts in a way that another server (wildfly, tomcat, etc.) does not start anymore.
